lib/screens/edit_profile_screen/edit_profile_screen.dart

```diff
@@ -159,6 +173,69 @@ class _EditProfileScreenState extends State<EditProfileScreen> {
                           SizedBox(
                             height: 16,
                           ),
+
+                          Text(
+                            AppLocalizations.of(context)!.links,
+                            style: GoogleFonts.roboto(
+                                fontSize: 16, fontWeight: FontWeight.w400),
+                          ),
+                          SizedBox(
+                            height: 16,
+                          ),
+                          GestureDetector(
+                            onTap: () async {
+                              dynamic list = await  Navigator.of(context)
```

Comment:
Pela linha 194 parece que sabe-se o retorno do pushNamed então trocar o dynamic pelo retorno correto se possível

Reply:
Tem que ser dynamic  pois é o que o Navigator.of retorna

Reply:
Alterado